tests/client_test.c

```diff
@@ -26,10 +26,10 @@
 #include <math.h>
 #include <signal.h>
 #ifdef VALKEY_TEST_SSL
-#include "valkey_ssl.h"
+#include "ssl.h"
```

Comment:
Minor:

In the examples we include `"valkey/ssl.h"` but in tests we write just `"ssl.h"`, because we use different compiler flags?

If we want the tests to be useful as examples, then maybe we should change them to `"valkey/ssl.h"` style too?

Reply:
The examples are only [built in CI](https://github.com/valkey-io/libvalkey/blob/main/.github/workflows/ci.yml#L102) by first installing libvalkey on the runner, and then build the examples using the installed headers and libs. So, here we use `<valkey/ssl.h>`. This is a verification job for `make install`.

But its a good idea, I'll take a stab at that in a separate PR.
The MakeList/Makefile needs a slight change and there will be a bunch of include updates.

Reply:
OK, or just ignore this if it's non-trivial.